textattack/commands/attack/attack_command.py

```diff
@@ -157,6 +157,24 @@ def str_to_int(s):
             default=float("inf"),
             help="The maximum number of model queries allowed per example attacked.",
         )
+        parser.add_argument(
```

Comment:
When would the user ever need to set the batch size when attacking? We don't process attacks in batches.